https://github.com/cython/cython/commit/bf4979e2441ffbc9aaeb88f5c67608578040588f
https://bugs.gentoo.org/739516

From: Stefan Behnel <stefan_ml@behnel.de>
Date: Sat, 3 Apr 2021 08:23:44 +0200
Subject: [PATCH] Make a helper function in a C++ test correctly propagate
 exceptions so that it won't have to spit out compiler warnings.

--- a/tests/run/cpp_stl_conversion.pyx
+++ b/tests/run/cpp_stl_conversion.pyx
@@ -15,7 +15,7 @@ py_set = set
 py_xrange = xrange
 py_unicode = unicode
 
-cdef string add_strings(string a, string b):
+cdef string add_strings(string a, string b) except *:
     return a + b
 
 def normalize(bytes b):

